src/Pages/Category/Component/ProductCardGrid/ProductCardGrid.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import ItemBoxSmall from "../ItemBoxSmall/ItemBoxSmall";
+import "./ProductCardGrid.scss";
+
+class ProductCardGrid extends Component {
+  render() {
+    const { productList } = this.props;
+    console.log(this.props.productList);
+    return (
+      <article className="ProductCardGrid">
+        {productList &&
+          productList.map((item, idx) => {
```

Comment:
이렇게 줄여볼 수 있습니다! Optional Chaining!
```suggestion
        {productList?.map((item, idx) => {
```

Reply:
수정했습니다 감사합니다!